processor/tailsamplingprocessor/config.go

```diff
@@ -97,6 +122,8 @@ type PolicyCfg struct {
 	RateLimitingCfg RateLimitingCfg `mapstructure:"rate_limiting"`
 	// Configs for defining composite policy
 	CompositeCfg CompositeCfg `mapstructure:"composite"`
+	// Configs for defining combined policy
+	CombinedCfg CombinedCfg `mapstructure:"combined"`
```

Comment:
We'd need to document well how it would work to have a Composite with a Combined. It might be very confusing to end-users. Perhaps we should not have it first, and implement only when there's concrete demand?

Reply:
Is this a left-over from a previous iteration?